Sources/Common/Core/PriorityQueue/index.js

```diff
@@ -0,0 +1,61 @@
+import macro from 'vtk.js/Sources/macro';
+
+// ----------------------------------------------------------------------------
+// vtkPriorityQueue methods
+// ----------------------------------------------------------------------------
+
+function vtkPriorityQueue(publicAPI, model) {
+  // Set our classname
+  model.classHierarchy.push('vtkPriorityQueue');
+
+  publicAPI.push = (priority, element) => {
+    // naive algo
+    model.elements.push({ priority, element });
+    model.elements.sort(
+      ({ priorityLeft }, { priorityRight }) => priorityLeft - priorityRight
```

Comment:
you need to be careful with the sort algo. It is unstable with old chrome. Please do something in case of equality to enforce an order.

Reply:
Ok.